master/marathon-leader.lua

```diff
@@ -0,0 +1,21 @@
+local cache = require "master.cache"
+
+local mleader = cache.get_cache_entry("marathonleader")
+if mleader == nil then
+    ngx.status = ngx.HTTP_SERVICE_UNAVAILABLE
+    ngx.say("503 Service Unavailable: unable to obtain Marathon leader cache")
```

Comment:
Of course, we can unify it - I did not want to change too many things in this already big PR.

I standardised on `503 Service Unavailable: cache state is invalid`.